randovania/games/dread/json_data/Cataris.txt

```diff
@@ -2308,13 +2308,17 @@ Extra - asset_id: collision_camera_031
   * Extra - right_shield_def: None
   > Dock to EMMI Zone Tower East
```

Comment:
While reviewing it looks like this doesn't contain a flash shift method.

Reply:
indeed, it is missing flash + intermediate wall jump, and morph + single wall-jump intermediate/advanced

Reply:
I honestly didn't even pay attention to the other missing logic here...there are so many ways to ADM skip! :P

I can add those two...I'm not sure what to use as a reference for deciding the difficulty of the SWJ with morph ball though. They're all difficult for me 😂